inc/upgrade.php

```diff
@@ -253,3 +310,59 @@ function responsive_upgrade_2_0( $verbose = true ) {
 		update_option( 'burf_setting_layout', $new_layout );
 	}
 }
+
+/**
+ * Migrate font options.
+ *
+ * @param boolean $verbose Flag for outptting messages to error log.
+ */
+function responsive_upgrade_fonts( $verbose ) {
+	if ( $verbose ) {
+		error_log( __FUNCTION__ . ' - Updating font.' );
+	}
+
+	$old_font = get_option( 'burf_setting_fonts' );
+	$new_font = 'f1';
+
+	if ( defined( 'BU_RESPONSIVE_FONT_PALETTE' ) ) {
+		$new_font = BU_RESPONSIVE_FONT_PALETTE;
+	} elseif ( ! empty( $old_font ) ) {
+		$new_font = $old_font;
+	}
+
+	if ( 'f1' !== $new_font ) {
```

Comment:
Some inline comments may be helpful here to understand why we want to check `if ( 'f1' !== $new_found )`